updating/updating-restricted-network-cluster.adoc

```diff
@@ -123,7 +135,27 @@ include::modules/update-service-install-cli.adoc[leveloffset=+2]
 
 include::modules/update-service-graph-data.adoc[leveloffset=+2]
 
-include::modules/update-service-mirror-release.adoc[leveloffset=+2]
+[id="update-service-mirror-release_{context}"]
+=== Mirroring the {product-title} image repository
+
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+//The module below is being used twice in this assembly, so this instance needs to have a unique context set in order for its ID to be unique. In the future, if and when the two main sections of this webpage are split into their own assemblies/pages, the context attributes below can be removed.
+
+:!context:
```

Comment:
Although this probably works, I'm not sure if this something that we want to allow. 

I'd recommend checking in with @kalexand-rh on whether this is an acceptable workaround, or if we even have any other assemblies that try to include the same module twice. Seems like something we should probably not be doing, but there might be other places that do.

I know we talked about that this assembly should likely have been broken up into two assemblies. I know you mentioned it in your comment above about it maybe being done in the future, but did you give any thought in to doing that now with this update?

Reply:
I've never actually seen it happen, but it's something allowed in and described by [mod docs](https://redhat-documentation.github.io/modular-docs/#reusing-modules). (Search for "Module included multiple times in a single assembly")

It would be better to split it into two assemblies now, but it is legit to use the module twice.

Reply:
I think it's the first time in the four years we've been doing mod docs that it's come up!